queries/fortran/highlights.scm

```diff
@@ -122,6 +127,8 @@
   ")"
   "["
   "]"
+  "<<<"
+  ">>>"
  ] @punctuation.bracket
```

Comment:
feels weird having these as bracket, maybe special? Not a fortran programmer, so.

Reply:
@oponkork

Reply:
Ah, it's for CUDA kernels. Then, it's like in C++